apps/fortuna/src/config.rs

```diff
@@ -172,6 +176,10 @@ pub struct EthereumConfig {
     pub priority_fee_multiplier_pct: u64,
 }
 
+fn default_backoff_gas_multiplier_pct() -> u64 {
+    100
```

Comment:
we may want to set this to some reasonable % in the future, but let's see how the abstract thing goes first.